src/i18n.ts

```diff
@@ -0,0 +1,21 @@
+import i18n from "i18next";
+import { initReactI18next } from "react-i18next";
+import LanguageDetector from "i18next-browser-languagedetector";
+import Backend from "i18next-http-backend";
+
+export enum Languages {
+    ru = "ru",
+    en = "en",
```

Comment:
enum keys should be capitalized
```suggestion
    Ru = "ru",
    En = "en",
```

Reply:
Fixed